### ISSUE TITLE
Default silenced views (CBV) does not with Django@3.2

### ISSUE THREAD

**Issue by DmytroLitvinov** [AliSayyah/django-urlconfchecks#132]
**Describe the bug**
After adding fine-grained method for silencing errors in corresponded PR https://github.com/AliSayyah/django-urlconfchecks/pull/126 there is a [default line for CBV views](https://github.com/AliSayyah/django-urlconfchecks/blob/5534e38fa71a1c268cc601f107097beeaddd7478/django_urlconfchecks/check.py#L17) which does not work for Django@3.2

**To Reproduce**
1. Setup Django@3.2 at your project.
2. Prepare SimpleView class:
```
class SimpleView(View):
    """
    A simple view with a docstring.
    """

    def get(self, request):
        return HttpResponse("This is a simple view")
```
3. Use that view as a CBV view at urls
4. Run `urlconfchecks`

**Expected behavior**
0 error

**Actual behavior**
1 error

Why this is happening? Starting from Django@4, using `__qualname__` will transform correctly according to a default one that is defined in that package. Here is unit test from Django@4 - https://github.com/django/django/blob/898f0aa44fe63c20d7f4125672b8eb800a578ce5/tests/generic_views/test_base.py#L189

For Django@3.2 there is no such unit test for `__qualname__
In Django==3.2 there is no such test: https://github.com/django/django/blob/777362d74aa3f3d88e6d60199f1d986200fb83b8/tests/generic_views/test_base.py#L173
And it simply returns the name of the class - just `SimpleView`

**Versions:**
 - Django==3.2
 - Python==3.8

**Additional context**
The first initial request of that bug was commented in next PR: https://github.com/AliSayyah/django-urlconfchecks/pull/126#issuecomment-1222001737

**Comment by AliSayyah** [comment]
@DmytroLitvinov Thank you for reporting this.
I'll work on a fix.

**Comment by DmytroLitvinov** [comment]
Thanks for taking this into account.

**Comment by AliSayyah** [comment]
@DmytroLitvinov Released v0.7.3 with a fix.
Can you please confirm that the issue is fixed?

**Comment by DmytroLitvinov** [comment]
Hi @AliSayyah ,
Thanks for letting me know.
Yes, it works.

If someone is using `djangorestframework`, you can add next line to customized dict:
`"rest_framework.views.APIView.as_view": "W001",`
